js/components/addressBookList/addressBookList_controller.js

```diff
@@ -7,6 +7,14 @@ angular.module('contactsApp')
 	AddressBookService.getAll().then(function(addressBooks) {
 		ctrl.addressBooks = addressBooks;
 		ctrl.loading = false;
+		if(ctrl.addressBooks.length === 0) {
+			AddressBookService.create(t('contacts', 'Contacts')).then(function() {
```

Comment:
Only one little enhancement would be nice: move this logic to the service, so that when other code paths use the service, the addressBook is created too :-)